<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockColorInput</class>
 <widget class="QWidget" name="BlockColorInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Primaries &amp; White point</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="BigComboBox" name="primariesBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Transfer curve</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="BigComboBox" name="trcBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="rangeLabel">
     <property name="text">
      <string>YUV Range</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="BigComboBox" name="rangeBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>YUV to RGB Matrix</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="BigComboBox" name="spaceBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BigComboBox</class>
   <extends>QComboBox</extends>
   <header>UI/bigcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
